sql/index_builder.go

```diff
@@ -54,6 +56,32 @@ func NewIndexBuilder(idx Index) *IndexBuilder {
 	}
 }
 
+func roundUpFloat(val interface{}) interface{} {
+	switch v := val.(type) {
```

Comment:
Last topic is whether there are other packages that have already implemented portions of this logic, which would be a signal for deduplication at some point, or places that might need to use this in the future, whether for constant folding or something. Tag the relevant areas (if any) with TODOs, and then chose the package for this logic based on what else might need to use it (the wrapper i suggested might be best placed in `types`?)